scripts/migrate.js

```diff
@@ -94,6 +95,22 @@ const run = async () => {
     rimraf.sync(NAV_DIR);
     rimraf.sync(DICTIONARY_DIR);
 
+    logger.info('Fetching redirects');
+
+    const orderedRedirects = (docs) => {
+      return docs.reduce((accum, curr) => {
```

Comment:
Nitpick: can we use `acc` rather than `accum`? I believe we use this convention with our other `reduce` usage.